bridge_adaptivity/module/views.py

```diff
@@ -675,6 +675,23 @@ class SequenceComplete(LtiSessionMixin, DetailView):
     template_name = 'module/sequence_complete.html'
 
 
+def _check_and_build_web_socket_message(sequence, score):
+    """
+    Build a dictionary with data for callback by Web Socket.
+
+    Check flags ui_option, congratulation_message and validate score.
+    """
+    web_socket_message_dict = {}
```

Comment:
I would create dict with the default key: value pair:
```
web_socket_message_dict = {"is_button_enable": True}
```

Reply:
It makes sense. I have done it.